pandas/core/algorithms.py

```diff
@@ -596,7 +596,11 @@ def _factorize_array(
 )
 @Appender(_shared_docs["factorize"])
 def factorize(
-    values, sort: bool = False, na_sentinel: int = -1, size_hint: Optional[int] = None
+    values,
+    sort: bool = False,
+    na_sentinel: int = -1,
+    size_hint: Optional[int] = None,
```

Comment:
can you add some independent tests for factorize with dropna

Reply:
emm, I think i added two test cases already in `test_algos`. Is it what you mean? or you want to have different test cases?